File: aws-lambda-in-action/ch02/custom-greetings-on-demand.py
import json

print('Loading function')

def lambda_handler(event, context):
    print('Received event: ' + json.dumps(event, indent=2))
    greet = event['greet'] if 'greet' in event else 'Hello'
    name = event['name'] if 'name' in event else 'World'
    greetings = greet + ' ' + name + '!'
    print(greetings)
    return greetings

File: aws-lambda-in-action/ch02/custom-greetings-on-demand.js
console.log('Loading function');

exports.handler = (event, context, callback) => {
    console.log('Received event: ', JSON.stringify(event, null, 2));
    var greet = ('greet' in event) ? event.greet : 'Hello';
    var name = ('name' in event) ? event.name : 'World';
    var greetings = greet + ' ' + name + '!';
    console.log(greetings);
    callback(null, greetings);
};